migrations/20240628155246_init-tables.up.sql

```diff
@@ -62,13 +64,13 @@ CREATE TABLE queues(
     properties JSONB,
     parent TEXT,
     template_info JSONB,
-    children JSONB,
-    children_names TEXT[],
     abs_used_capacity JSONB,
     max_running_apps INTEGER,
     running_apps INTEGER NOT NULL,
     current_priority INTEGER,
     allocating_accepted_apps TEXT[],
+    created_at BIGINT NOT NULL,
```

Comment:
Can you move these two new columns up, so they are right after `parent_id` (and before `queue_name`)  - I expect we will have a number of queries against these, and there is a slight performance improvement when a column has only fixed-width-type columns before it (no TEXT or JSONB, etc).